### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]:"Browse" and "Refresh" buttons  are not accessible with keyboard in Run on web app containers window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1440]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app for containers.
4. "Run on web app for Containers" dialog will open.
5. Navigate and try to access the  "Browse" button "Refresh" button with keyboard.

**Note:** Same issue is observed in create web app in Run on web app containers window.

**Expected Result:**
"Browse"  and "Refresh" buttons should be accessible with keyboard in Run on web app containers dialog.

**Actual Result:**
"Browse"  and "Refresh" buttons  are not accessible with keyboard in Run on web app containers dialog.
**Note:** User able to access the buttons by using below keys:
Before Launch -'Alt+b'
Add- 'Alt+insert'
Remove- 'Alt+Delete'
Edit- 'Enter'
Up- 'Alt+Up'
Down- 'Alt+down'.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![buttons not accessible in run on web app containers window](https://user-images.githubusercontent.com/22956130/35728406-585a5fdc-07c0-11e8-84ce-a8ef61a5c25e.png)

**Comment by Eskibear** [comment]
Browse button in widget "com.intellij.openapi.ui.TextFieldWithBrowseButton" cannot get tab focus.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191311

**Comment by Eskibear** [comment]
Buttons in decorator cannot get tab focus either.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191322

**Comment by Eskibear** [comment]
External issues tracked in #1627

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749900
